tslint.json

```diff
@@ -4,31 +4,14 @@
         "max-line-length": {
             "options": [120]
         },
-        "array-bracket-spacing": [2, "never"],
-        "comma-dangle": ["error", "always-multiline"],
-        "computed-property-spacing": [2, "never"],
-        "curly": 2,
-        "eqeqeq": [2, "smart"],
-        "indent": [
-          2,
-          2,
-          {
-            "SwitchCase": 1
-          }
-        ],
-        "max-depth": [1, 3],
-        "max-len": [1, 80],
-        "max-statements": [1, 15],
-        "no-use-before-define": [2, "nofunc"],
-        "object-curly-spacing": [2, "never"],
+        "trailing-comma": [true, {"multiline": "always"}],
```

Comment:
I'd just want to understand - is this a set of rules we are using in other TypeScript libraries?

Reply:
it depends - we still don't have a standard for this. In the last TypeScript project @gjvis decided to leave it almost blank with:
```json
{
    "defaultSeverity": "error",
    "extends": [
        "tslint:recommended"
    ],
    "jsRules": {},
    "rules": {
        "quotemark": {
            "options": "single"
        }
    },
    "rulesDirectory": []
}
```

in this PR, I tried to leave the original intension